ajax/libs/bootstrap3-wysihtml5-bower/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "bootstrap3-wysihtml5-bower",
+  "version": "0.3.3",
+  "filename": "bootstrap3-wysihtml5.all.min.js",
+  "description": "Bootstrap 3 compatible wysiwyg editor",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Waxolunist/bootstrap3-wysihtml5-bower.git"
```

Comment:
As **this** repo`(bootstrap-wysiwyg/bootstrap3-wysiwyg)` is a copy from `"Waxolunist/bootstrap3-wysihtml5-bower"`, and they are really preparing for next version. I think "repository" and "homepage" field could change to  **this** repo.